client/src/app/(main)/track/page.tsx

```diff
@@ -7,19 +7,14 @@ import NewApplicationModal from "@/ui/track/NewApplicationModal";
 import { DragDropContext, DropResult } from "@hello-pangea/dnd";
 import { StageColumn, StageColumnProps } from "@/ui/track/StageColumn"; 
 import { Application, ApplicationStage, useTrack } from "@/lib/store/track";
-import { useProfile } from "@/lib/store/profile";
-import { AlertMessage } from "@/ui/AlertMessage";
+import { useProfile } from "@/lib/store/profile"; 
 import "./tracking.css";
 
 export default function Page() {
   const [openModal, setOpenModal] = useState(false);
   const [applicationStatus, setApplicationStatus] = useState<ApplicationStage>("Applied");
   const [selectedApplication, setSelectedApplication] = useState<Application | undefined>();
-  const [snackbarOpen, setSnackbarOpen] = useState(false);
-  const [snackbarMessage, setSnackbarMessage] = useState("");
-  const [snackbarSeverity, setSnackbarSeverity] = useState<"success" | "error" | "info" | "warning">("success");
-
-
+  const [preventClick, setPreventClick] = useState(false);
```

Comment:
is this meant to handle the click for the modal?

Reply:
Yes, since I will have two modal (one is for application and one delete application prompt) the issue was whenever I clicked on the card, it would pop up the application modal after the popping up the delete promt. I have to adjust the logic between both.